README.md

```diff
@@ -39,6 +39,7 @@
   - [Include Parent Namespaces](#include-parent-namespaces)
   - [Include Missing](#include-missing)
   - [Evaluate Given](#evaluate-given)
+  - [Parameters precedence](#parameters-precedence)
```

Comment:
It should be "Parameter Precedence" (singular, capitalize both words)

Reply:
Updated.